### DISCUSSION TITLE
How to completely remove?

### DISCUSSION THREAD

**Discussion by msineath** [blitz-js/blitz#3261] [Q&A]
Not all projects I work with need blitz, but when I try to run next build on a non blitz project, it gives me an error saying to change next.config to blitz.config and refuses to build, even after uninstalling globally and clearing the package cache. Had blitz not interfered with my other projects, I would have used it, but forcing anything next to require blitz is bordering on malicious. How do I remove the blitz checker?

**Comment by noxify** [comment]
Correct me if I'm wrong, but wouldn't it be easier to use nextjs directly for the use cases where you don't need blitz? 

Maybe you can explain why you want to Setup your project with blitz and then removing blitz from the project?

If I misunderstood your question, sorry for that ✌️

**Reply by msineath** [reply]
I'm not trying to remove blitz from the blitz project, I'm trying to open a different project in a different folder and when I run next build, the terminal says blitz wants me to change next.config to blitz.config and refuses to build. Blitz was installed globally (as per the docs' instructions), so I uninstalled blitz, same issue. I cleared the npm cache, same issue. I just want blitz to stop trying to force itself on anything next related in other projects.

UPDATE: I restarted VS Code and that fixed everything. I guess a restart is in order after you remove blitz globally?
